### PR TITLE
[kai/delete-address-table] Delete address table

### PR DESCRIPTION
**Pull request by kai-wei-mo**
## Notion ticket link

[Remove address table and references](https://www.notion.so/uwblueprintexecs/Remove-address-table-and-references-f905caf33cc4482bb8a7dfae291276f2)

## Implementation description
* Delete the `address` model and table
* as well as anything that references/uses `address`
* Now, `address` is a `string` field instead of a `fk`

## Steps to test
1. i believe unit tests and e2e tests should be enough

## What should reviewers focus on?
* Correctness of alembic migration
* everything else has more-or-less the same level of focus priority ¯\\_(ツ)_/¯

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Review by jennchenn — CHANGES_REQUESTED** [review]
getting some errors with running the migration but otherwise looks like you've found most of the references to the address table!

File: backend/python/app/resources/daytime_contact_dto.py [hunk 1]
```diff
@@ -1,16 +1,19 @@
+import string
```
[the remark this answers is not in the document]
Comment (kai-wei-mo) [reply backend/python/app/resources/daytime_contact_dto.py]: i have no idea how that got there LOL

File: backend/python/migrations/versions/2022-11-01_967df97c58f0_remove_address_table.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
+"""remove address table
```
[the remark this answers is not in the document]
Comment (kai-wei-mo) [reply backend/python/migrations/versions/2022-11-01_967df97c58f0_remove_address_table.py]: should be fixed now, thanks!

File: backend/python/tools/db_seed.py [hunk 3]
```diff
@@ -54,17 +54,6 @@ def insert_test_data():
     for value in values:
```
[the remark this answers is not in the document]
Comment (kai-wei-mo) [reply backend/python/tools/db_seed.py:54]: fixed now, thanks!

**Review by jennchenn — APPROVED** [review]
upgrade works now, nice work!!!

**Pull request merged**